docs/helm.md

```diff
@@ -8,15 +8,52 @@ Tilt supports Helm out-of-the-box.
 
 There are two ways you can use Helm charts with Tilt.
 
-## Basic Helm
+- Install a remote Helm chart from the rich library of existing charts.
 
-If you have a local Helm chart and want to deploy it as-is, add a `helm()` call to your `Tiltfile`:
+- Iterate on a local Helm chart that you're building.
 
-```python
-k8s_yaml(helm('path/to/chart/dir'))
+Let's dig into each one.
+
+## Remote charts
+
+The `helm_remote` extension lets you load from a library of existing charts.
+
+First, add `.helm` to the `.tiltignore` file next to your `Tiltfile`. The
```

Comment:
wdyt about adding `watch_settings(ignore='.helm')` to the extension, so we can drop this step?

Reply:
done